crates/ruma-events/src/call.rs

```diff
@@ -10,6 +10,7 @@ pub mod invite;
 pub mod member;
 pub mod negotiate;
 #[cfg(feature = "unstable-msc4075")]
+#[deprecated = "Since Matrix 0.30.6. Use the m.rtc.notification event instead."]
```

Comment:
No "since" needed since it's unstable and was never part of a Matrix release.

Also I don't recommend putting the whole module as deprecated, it behaves weirdly with tests. You can just put the struct as deprecated.